demo/src/main/java/com/novoda/storagepathfinder/demo/StoragePathsAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.novoda.storagepathfinder.demo
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.novoda.storagepathfinder.StoragePath
+
+typealias Listener<T> = (T) -> Unit
```

Comment:
Thought saying it was a listener was more obvious than having `StoragePath -> Unit` when passing the param for a listener.